vscode_extension/src/AacLanguageServer.ts

```diff
@@ -1,9 +1,17 @@
 import * as fs from "fs";
-import { ExtensionContext, workspace } from "vscode";
-import { LanguageClient, LanguageClientOptions, ServerOptions, Trace } from "vscode-languageclient/node";
+import * as net from "net";
+
+import { ExtensionContext, window, workspace } from "vscode";
+import { LanguageClient, LanguageClientOptions, ServerOptions, StreamInfo, Trace } from "vscode-languageclient/node";
 import { assertTrue, execShell, getConfigurationItem, showMessageOnError, getSemanticVersionNumber } from "./helpers";
 
+enum LspServerMode {
```

Comment:
I don't like this pattern but we can't get these values from the configuration information provided by the VSCode API (at least I haven't found a way to do it) so I created story #303 to give us more information about the configuration such as possible values when we have [enumerated values](https://github.com/jondavid-black/AaC/blob/cbb06b08525caa94af093263bdcb3fb52be5183a/vscode_extension/package.json#L52-L55) a user can pick.